extensions/ql-vscode/src/view/common/ActionButton/ActionButton.css

```diff
@@ -0,0 +1,61 @@
+/* Styles have been copied from https://vscode-elements.github.io/elements-lite/components/action-button/configurator/ */
+.vscode-action-button {
```

Comment:
I think it would be nice if we were able to use CSS modules for this instead of adding global CSS classes. That ensures that there are no conflicts and we aren't overriding any styles. I don't think we can currently use CSS modules, but I can look into whether we can use those in the extension and create a PR to set it up if we can.

Reply:
https://github.com/github/vscode-codeql/pull/4000